.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,14 @@
 * **Please check if the pull request fulfills these requirements**
 - [ ] Tests for the changes have been added (for bug fixes / features)
+  - [ ] Does it pass all existing unit tests without modification?
+    - If not, what did you change?
+    - If you altered it significantly, what coverage issue did you fix?
+  - [ ] Does it compile & pass all tests in [every supported version](../CONTRIBUTING.md#unity--lts-support)?
```

Comment:
I get what you are going with here but I get the feeling this will either result in people not filing a PR because they are concerned they didn't test it over five versions of Unity OR just ignoring this part of the form.